apps/api/src/domain/common/events/in-memory-event-publisher.ts

```diff
@@ -1,71 +0,0 @@
-import { Inject, OnModuleDestroy } from '@nestjs/common';
```

Comment:
Note that as elegant as this was, RxJS proved really hard to debug, consistent with its reputation. We use a simpler to reason about `SyncInMemoryEventPublisher`, although, eventually we'll use a proper messaging queue.